robot/web/web_c.go

```diff
@@ -11,13 +11,13 @@ import (
 
 	"github.com/edaniels/golog"
 	"github.com/pkg/errors"
-	"github.com/viamrobotics/gostream"
-	streampb "github.com/viamrobotics/gostream/proto/stream/v1"
 	"go.viam.com/utils"
 	"go.viam.com/utils/rpc"
 
 	"go.viam.com/rdk/components/audioinput"
 	"go.viam.com/rdk/components/camera"
+	"go.viam.com/rdk/gostream"
+	streampb "go.viam.com/rdk/gostream/proto/stream/v1"
```

Comment:
We did a lot of work to move all the proto files out of RDK and into API, and I think this needs to be moved over there as well. Otherwise we're adding tech debt here, as this proto needs to be managed, generated, imported, and versioned cleanly, and we no longer have any of the makefile targets or CI automation in place in RDK for that.

Reply:
Done: https://github.com/viamrobotics/api/pull/388